### ISSUE TITLE
Error on https://www.albinoblacksheep.com/games/nanaca

### ISSUE THREAD

**Issue by Fips-e** [ruffle-rs/ruffle#4442]
# Error Info
Error name: Error
Error message: panicked at 'already borrowed: BorrowMutError', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:62:29
Error stack:
```
Error: panicked at 'already borrowed: BorrowMutError', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:62:29
    at a.wbg.__wbg_new_df6e6ab7a65c4c4d (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/ruffle.js:1:6349)
    at std::sync::once::Once::call_once::{{closure}}::h7141d078b1ea013f (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[1267]:0x24ef2a)
    at std::sync::once::Once::call_inner::h02eac160d339afc2 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[1528]:0x27ed9a)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hf48d3d3ee3d10801 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[4308]:0x353f1a)
    at std::panicking::rust_panic_with_hook::h123718ba3bf480af (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[3299]:0x32cc21)
    at std::panicking::begin_panic_handler::{{closure}}::hf393a82e58397bd2 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[3916]:0x349dfc)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h2ff2cfc953878925 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[4654]:0x359654)
    at rust_begin_unwind (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[4525]:0x357c32)
    at core::panicking::panic_fmt::h285dc015a87be5a4 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[4656]:0x3596b8)
    at core::option::expect_none_failed::hb95d2b1dc5448c17 (https://www.albinoblacksheep.com/script/ruffle_nightly_2021_01_26_selfhosted/aafdd926c74ff1ff1e03a02091c0405b.wasm:wasm-function[3995]:0x34c594)
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: /swf/games/nanaca(www.albinoblacksheep.com).swf
Param movie: /swf/games/nanaca(www.albinoblacksheep.com).swf
Param quality: high
Param menu: true
Param wmode: window
Param allowFullScreen: true
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined

# Page Info
Page URL: https://www.albinoblacksheep.com/games/nanaca
SWF URL: https://www.albinoblacksheep.com/swf/games/nanaca(www.albinoblacksheep.com).swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-26
Channel: nightly
Built: 2021-01-26T00:36:48.131Z
Commit: 7856cdd3d770e406ce546bf1745d8046d7b716f8

**Comment by shinstc** [comment]
Duplicate of #3246, crash fixed by #3322 (a very old version of Ruffle is being used on this site).

**Comment by torokati44** [comment]
That's probably right; thanks for checking and commenting!
